yacc.go

```diff
@@ -3470,22 +3469,20 @@ func $$Parse($$lex $$Lexer) int {
 
 func ($$rcvr *$$ParserImpl) Parse($$lex $$Lexer) int {
 	var $$n int
-	var $$lval $$SymType
 	var $$VAL $$SymType
 	var $$Dollar []$$SymType
 	_ = $$Dollar // silence set and not used
-	$$S := make([]$$SymType, $$MaxDepth)
+	$$S := $$rcvr.stack[:]
```

Comment:
not sure why this change; if `ParserImpl` is never reused, it seems equivalent to not have made this change at all.